contentcuration/contentcuration/frontend/shared/views/StudioChip.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+
+  <div
```

Comment:
These buttons when the whole area is clickable need to be `<button>` - to ensure proper semantics.

<img width="480" height="81" alt="Screenshot from 2025-10-28 12-16-36" src="https://github.com/user-attachments/assets/d81551e3-ef7d-4c14-abc9-db68c035c938" />